packages/uswds-core/src/styles/settings/_settings-utilities.scss

```diff
@@ -28,7 +28,8 @@ used by utilities or layout grid
 ----------------------------------------
```

Comment:
I am wondering if it would be helpful if we added a note here that users should edit `$theme-utility-breakpoints` in the settings configuration and not `$theme-utility-breakpoints-complete`.